### DISCUSSION TITLE
About SofaSDKForUnity Report an error

### DISCUSSION THREAD

**Discussion by Lihao7788** [sofa-framework/sofa#4287] [Write / Run a simulation]
I used the SDK of SOFAForUnity to enter the following scene, but this scene is running, and there is no effect, the following is my scene and error screenshot, please help me, we are about to use SOFA cutting function, thank you very much.

![Uploading 图片1.png…]()
![Uploading 图片2.png…]()

**Comment by Lihao7788** [comment]
Hello After I entered the scene, the SofaContext script was automatically disabled, below is his error message
ExecutionEngineException: String conversion error: Illegal byte sequence encounted in the input.
SofaUnityAPI.SofaContextAPI.DisplayMessages () (at Assets/SofaUnity/Plugins/SofaUnityAPI/SofaContextAPI.cs:429)
SofaUnity.SofaContext.DoCatchSofaMessages () (at Assets/SofaUnity/Scripts/SofaContext.cs:564)
SofaUnity.SofaContext.Update () (at Assets/SofaUnity/Scripts/SofaContext.cs:433)

**Comment by epernod** [comment]
Hi,

same, could you open a discussion into this repository: https://github.com/InfinyTech3D/SofaUnity-Renderer/discussions
to keep the record.

Also I think there is an issue with the images you uploaded, I can't see them. Could you also check there is no more error message in the console before the string issue?

Thank you
